src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -85,7 +94,32 @@ describe("calcite-input-time-picker", () => {
     expect(await input.getProperty("value")).toBe("");
   });
 
-  it("programmatically changing the value reflects in the input for 24-hour (french lang)", async () => {
+  it("opens the time picker on input keyboard focus", async () => {
+    const page = await newE2EPage({
```

Comment:
For consistency and test stability, can you create the page and then use `page.setContent`?

Applies to other tests.